### PR TITLE
Added partitions and replicationFactor options to createCustomTopic

### PR DESCRIPTION
**Pull request by chris-zen**

### CODE REVIEW

File: src/main/scala/net/manub/embeddedkafka/EmbeddedKafka.scala [hunk 1]
```diff
@@ -246,11 +246,13 @@ sealed trait EmbeddedKafkaSupport {
     * @param topicConfig per topic configuration [[Map]]
     * @param config      an implicit [[EmbeddedKafkaConfig]]
     */
```
Comment (manub) [inline src/main/scala/net/manub/embeddedkafka/EmbeddedKafka.scala]: Can you please add some ScalaDocs here?

**Comment by manub** [comment]
It would be good also to update the `README` file with those changes.

**Comment by chris-zen** [comment]
@manub I updated the ScalaDocs and the README, please let me know if there is anything else that need to be addressed ;-)

**Comment by manub** [comment]
Planning to release this during the weekend.

**Pull request merged**